mappings/net/minecraft/world/gen/feature/OceanRuinFeature.mapping

```diff
@@ -0,0 +1,29 @@
+CLASS net/minecraft/unmapped/C_zjruaixz net/minecraft/world/gen/feature/OceanRuinFeature
+	FIELD f_djmsjras CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_dxrhwnfn clusterProbability F
+	FIELD f_mplzpvss largeProbability F
```

Comment:
Is large here referring to size of the ruin? Or is it an adjective for "probability"? I would think the first, but this name is still ambiguous nonetheless